src/Sulu/Bundle/MediaBundle/Resources/public/js/components/media-selection/main.js

```diff
@@ -414,33 +483,192 @@ define(['sulumedia/collection/collections', 'sulumedia/model/collection'], funct
                     name: 'overlay@husky',
                     options: {
                         triggerEl: this.$addButton,
-                        cssClass: 'media-selection-overlay',
-                        el: $element,
+                        draggable: false,
                         removeOnClose: false,
+                        el: $element,
                         container: this.$el,
-                        draggable: false,
-                        instanceName: 'media-selection.' + this.options.instanceName + '.add',
-                        skin: 'medium',
+                        cssClass: 'media-selection-overlay-container',
+                        instanceName: this.options.instanceName + '.add',
+                        skin: 'wide',
+                        supportKeyInput: false,
                         slides: [
                             {
                                 title: this.sandbox.translate(this.options.translations.addImages),
-                                okCallback: getAddOverlayData.bind(this),
-                                cssClass: 'media-selection-overlay-add',
-                                tabs: [
+                                data: templates.mediaSelection({
+                                    contentDefaultTitle: this.sandbox.translate('media-selection.overlay.all-images')
+                                })
+                            }
+                        ]
+                    }
+                }
+            ]);
+
+            this.sandbox.on('husky.overlay.' + this.options.instanceName + '.add.opened', function() {
+                var selectedItems = this.getData().ids || [];
+                this.sandbox.emit('husky.datagrid.media-selection-ovelay.' + this.options.instanceName + '.selected.update', selectedItems);
+            }.bind(this));
```

Comment:
Wouldn't it be a good idea to keep this listeners in the `bindCustomEvents` function?